fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,42 +1,42 @@
-<b>WordPress for Android</b> puts the power of web publishing in your pocket. It’s a website creator and so much more!
+WordPress for Android puts the power of web publishing in your pocket. It’s a website creator and so much more!
 
-<b>CREATE</b>
-- Give your big ideas a home on the web. WordPress for Android is a website builder and a blog maker. Use it to create your website. 
-- Pick the right look and feel from a wide selection of WordPress <b>themes</b>, then customize with photos, colors, and fonts so it’s uniquely you.
-- Built-in <b>Quick Start</b> tips guide you through the setup basics to set your new website up for success. (We’re not just a website creator -- we’re your partner and cheering squad!)
+CREATE
+- Give your big ideas a home on the web. WordPress for Android is a website builder and a blog maker. Use it to create your website.
+- Pick the right look and feel from a wide selection of WordPress themes, then customize with photos, colors, and fonts so it’s uniquely you.
+- Built-in Quick Start tips guide you through the setup basics to set your new website up for success. (We’re not just a website creator -- we’re your partner and cheering squad!)
```

Comment:
Those changes (i.e. removal of the tags) are simply because those tags were never there in the GlotPress originals (and translations) in the first place, so this version without tags is actually how the copies have been looking all those times in GlotPress (and PlayStore) in reality.